scripts/release-changelog.sh

```diff
@@ -25,5 +25,6 @@ github_changelog_generator -u fastly -p go-fastly \
   --bugs-label "**Bug fixes:**" \
   --release-url "https://github.com/fastly/go-fastly/releases/tag/%s" \
   --exclude-labels documentation \
+  --exclude-tags-regex "v.*-.*" \
```

Comment:
Makes what is used by `make changelog`

https://github.com/fastly/go-fastly/blame/936776b61a119658c8216a931e91d5678245d2fc/scripts/changelog.sh#L28